fan_control/fan_control.py

```diff
@@ -17,32 +18,35 @@
 fan = gpiozero.PWMOutputDevice(FAN_PIN, active_high=True, frequency=100)
 fan.value = 0
 
-# Returns PWM duty cycle
+
 def pwm_curve(temp):
+    ''' Returns PWM duty cycle '''
     if(temp < FAN_ON_TEMP_C):
         return 0
     else:
         return (1 - (PI_MAX_TEMP_C - temp) / (PI_MAX_TEMP_C - FAN_ON_TEMP_C))
 
-# Check fan temp + change pwm if needed
+
 def fan_control():
-  # Get CPU temp
-  temp = subprocess.run(["vcgencmd", "measure_temp"], stdout = subprocess.PIPE)
+    ''' Get CPU temp + change fan PWM as needed '''
```

Comment:
@AveryChiu64 changed comments to this.  I actually think this isn't _entirely_ with convention, but not having a space between the ''' and the text looks weird to me.  Lmk if this convention works for you

Reply:
This looks good